tests/test_schema/test_schema_logic.py

```diff
@@ -6,6 +6,7 @@
 
 import pytest
 
+from fmu.dataio._definitions import ALLOWED_CONTENTS
 
```

Comment:
Note to self: we should consier making `ALLOWED_CONTENTS` a Enum class in fmu-dataio